### ISSUE TITLE
Slave Market Gender Issue

### ISSUE THREAD

**Issue by hetryon** [Innoxia/liliths-throne-public#1714]
**Describe the bug**
In the gender-specific areas of the Slave Market (Iron&Steel and AWoman'sTouch), we've got girls in the boys and boys in the girls. The slaves themselves seem normal, it's just that they're where they're not supposed to be. 

**Expected behavior**
Functional gender separation in the gender-separated areas of the slave market. 

**To Reproduce**
1. Go to 'Iron & Steel' or 'A Woman's Touch'.
2. Observe females in 'Iron & Steel' and males in 'A Woman's Touch'. 
4. That's it, that's the error. 

**Attach files**
![BugReport1](https://user-images.githubusercontent.com/12883540/193708681-af308273-e61c-4685-8ae9-f1d18bdabff7.png)

**Additional information:**
 - Game version 0.4.6a
 - OS: Windows 10
 - Using .exe

**Comment by AceXP** [comment]
The slave market shops have slaves based on their femininity/masculinity, not their genital configuration. This is not likely to be changed.

**Comment by CognitiveMist** [comment]
Yes, this is by design. Pink name -> pink shop. Blue name -> blue shop.
